.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
+- repo: https://github.com/ambv/black
+  rev: 24.8.0
+  hooks:
+  - id: black
+    language_version: python3.9
```

Comment:
We are using python 3.10 so we need to change this + declare somewhere the python version we are using (ex. inside pipfile).